### PR TITLE
Docs: add text formatting rules

### PR DESCRIPTION
**Pull request by imatwawana**
Adds guidance for formatting abstract UI concepts and bold formatting in tables

### CODE REVIEW

File: docs/sources/write/style-guide/capitalization-punctuation/index.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ If you're not sure of whether something is a product or not, consult the product
   - Lowercase _dashboard_ when referring to a dashboard generally.
   - Lowercase _dashboards_ when referring to multiple dashboards.
 - **Exceptions:** If a term is lowercase in the Grafana UI, then match the UI.
+- Abstract UI concepts such as roles (Admin, Editor, Viewer) when you're not directly referring to the UI.
```
Comment (imatwawana) [inline docs/sources/write/style-guide/capitalization-punctuation/index.md:50]:
```suggestion
- Abstract UI concepts, such as roles (Admin, Editor, Viewer), when you're not directly referring to the UI.
```

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 2]
```diff
@@ -291,10 +291,11 @@ It's a good idea to take a consistent approach to bold, italic, and other text f
 
 ### Bold
 
-Use bold formatting (`**`), for UI elements.
-Don't use bold to draw attention to a word or phrase within a sentence, instead use italic emphasis.
+Use bold formatting (`**`) when directly referring to UI elements.
+When referring to an abstract UI element such as a role (Admin, Editor, Viewer) without directly referencing the UI, capitalize, but don't bold the word.
```
Comment (imatwawana) [inline docs/sources/write/style-guide/style-conventions/index.md:295]:
```suggestion
When you're referring to an abstract UI element, such as a role (Admin, Editor, Viewer), without directly referencing the UI, don't bold the word; just capitalize it.
```

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 3]
```diff
@@ -291,10 +291,11 @@ It's a good idea to take a consistent approach to bold, italic, and other text f
...
+It's okay to use bold inline with other prose in a table, but don't use bold for the all text in the leading column, even if those items are UI elements.
```
Comment (imatwawana) [inline docs/sources/write/style-guide/style-conventions/index.md:296]:
```suggestion
It's okay to use bold inline with other prose in a table, but don't use bold for the all text in the first column, even if it's a table of UI elements.
```

File: docs/sources/write/style-guide/style-conventions/index.md [hunk 4]
```diff
@@ -291,10 +291,11 @@ It's a good idea to take a consistent approach to bold, italic, and other text f
...
 ### Bold
 
-Use bold formatting (`**`), for UI elements.
-Don't use bold to draw attention to a word or phrase within a sentence, instead use italic emphasis.
+Use bold formatting (`**`) when directly referring to UI elements.
+When you're referring to an abstract UI element, such as a role (Admin, Editor, Viewer), without directly referencing the UI, don't bold the word; just capitalize it.
+It's okay to use bold inline with other prose in a table, but don't use bold for the all text in the first column, even if it's a table of UI elements.
 
-It's OK to use bold for a the first sentence in unordered lists that are followed by more information.
+It's okay to use bold for a the first sentence in unordered lists that are followed by more information.
```
Comment (jdbaldry) [inline docs/sources/write/style-guide/style-conventions/index.md:298]: Battle of the OK commences :D https://en.wikipedia.org/wiki/OK

I'm OK with either, with a preference for OK even though I guess it's a bit weird looking.

Google style doesn't have a documented opinion in their word list that I can find but seems to use okay in their sentences.

Comment (jdbaldry) [reply docs/sources/write/style-guide/style-conventions/index.md:298]: So perhaps we should say okay is the way and I'll stop adding more OKs to our docs

**Review by jdbaldry — APPROVED** [review]
LGTM

**Pull request merged**